TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-*"
```

Comment:
I agree, we should still be doing full builds, and just publishing to a separate location.  We can even use the same storage account, and just switch the blob container.

https://github.com/dotnet/core-setup/blob/release/1.0.0/build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs#L21